.github/workflows/smoke-test-docker.yml

```diff
@@ -14,4 +14,4 @@ jobs:
     uses: verdaccio/verdaccio/.github/workflows/x-smok-test-docker.yml@master
     with:
       docker_tag: '6.x-next'
```

Comment:
it should use 6.x-next because is updated on each commit on 6.x branch
https://hub.docker.com/layers/verdaccio/verdaccio/6.x-next/images/sha256-49940e3d2c8259c548cb73260810d5a201556b084e438a8fcf7cdfffe5d7121d